pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManagerTest.java

```diff
@@ -90,7 +91,7 @@
   static final long START_TIME_MS = CURRENT_TIME_MS - TimeUnit.HOURS.toMillis(RANDOM.nextInt(24) + 24);
   static final long END_TIME_MS = START_TIME_MS + TimeUnit.HOURS.toMillis(RANDOM.nextInt(24) + 1);
   static final Interval INTERVAL = new Interval(START_TIME_MS, END_TIME_MS);
-  static final String CRC = Long.toString(RANDOM.nextLong());
+  static final String CRC = Long.toString(RANDOM.nextLong() & 0xFFFFFFFFL);
```

Comment:
curious why `& 0xFFFFFFFFL` here?

Reply:
Added comment: CRC is always non-negative